PyTorch/SpeechSynthesis/FastPitch/common/utils.py

```diff
@@ -67,12 +67,15 @@ def load_wav_to_torch(full_path, force_sampling_rate=None):
     return torch.FloatTensor(data.astype(np.float32)), sampling_rate
 
 
-def load_filepaths_and_text(fnames, dataset_path=None, has_speakers=False,
+def load_filepaths_and_text(fnames, dataset_path=None, has_speakers=False, has_conditions=False,
                             split="|"):
     def split_line(line, root=None):
         parts = line.strip().split(split)
-        if has_speakers:
+        #need to add option for all conditions
+        if has_speakers or has_conditions: #this might be wrong @Johannah
```

Comment:
True or True = True
is this to say, the last item is either the speaker, or the condition?
given they're both booleans you could do:
`has_speakers is not has_conditions`
or put `has_speakers and has_conditions` first, so that this case is already handled and therefore excluded from``A or B'